advanced/sr_gui_cyberglove_calibrator/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sr_gui_cyberglove_calibrator)
-find_package(catkin REQUIRED COMPONENTS rospy rqt_gui rqt_gui_py std_msgs std_srvs diagnostic_msgs sr_robot_msgs sr_visualization_icons)
+find_package(catkin REQUIRED COMPONENTS rospy rqt_gui rqt_gui_py std_msgs std_srvs diagnostic_msgs sr_robot_msgs sr_visualization_icons python-scipy)
```

Comment:
I think this whole PR was agreed to not be merged but rather use mine including the fixes and some more https://github.com/shadow-robot/sr-visualization/pull/156